app/uk/gov/hmrc/overseaspensiontransferstubs/controllers/HipController.scala

```diff
@@ -34,16 +34,29 @@ class HipController @Inject() (
     resourceService: ResourceService
   ) extends BackendController(cc) with TimeDateHelpers with Logging {
 
-  def submitTransfer: Action[JsValue] = checkHeaders(parse.json) {
+  def submitTransfer(userId: String): Action[JsValue] = checkHeaders(parse.json) {
     _ =>
-      def getRandomFormBundle: String = Random.nextLong(999999999999L).toString
-      def getRandomQtNumber: String   = s"QT${100000 + Random.nextInt(900000)}"
+      if (userId == "A2100060") {
```

Comment:
In the Json payload we can read this value from qtDeclaration \ submitterId. Which would mean we don't need the parameter and don't need the to make a change in the backend.